test/integration/targets/test_infra/runme.sh

```diff
@@ -21,3 +21,7 @@ echo "rc was $APB_RC (must be non-zero)"
 echo "ensure playbook output shows assert/fail works (True)"
 echo "$PB_OUT" | grep -F "fail works (True)" || exit 1
 echo "$PB_OUT" | grep -F "assert works (True)" || exit 1
+
+
+PING_MODULE_PATH="$(pwd)/../../../../lib/ansible/modules/system/ping.py"
+../../../../hacking/test-module -m "$PING_MODULE_PATH"
```

Comment:
The tests are failing because freebsd doesn't have `/usr/bin/python`, so you will need to adjust this to use something like:

```
../../../../hacking/test-module -m "$PING_MODULE_PATH" -I ansible_python_interpreter=$(which python)
```

Reply:
All right, I will try this, thanks!